### PR TITLE
Enable tenant flags, session lifetimes and cookie.

### PR DESCRIPTION
**Pull request by czka**

### CODE REVIEW

**Comment by czka** [comment]
I'm not 100% sure whether I've used `lookup` in ae8dec01a1b34548707a478cc13c000ec9eebf5a correctly. Please decide if it's better to use `each.value.` for any of these 4 arguments.

If `lookup` is the right way for `flags`, should an empty map `{}` be the default for that arg?

File: variables.tf [hunk 1]
```diff
@@ -60,8 +60,8 @@ variable "tenant" {
   default = [{
     friendly_name           = ""
     default_redirection_uri = ""
-    # idle_session_lifetime   = "0"
-    # session_lifetime        = "0"
+    idle_session_lifetime   = "0"
+    session_lifetime        = "0"
```
Comment (czka) [inline variables.tf]: Shouldn't these 2 `"0"`'s actually be `72` and `168`, respectively, as per https://auth0.com/docs/get-started/tenant-settings (i.e. 3 and 7 days)? At least these seem to be the actual defaults the Auth0 `auth0_tenant` resource uses. Please double-check.

And in general why have exactly these 6 arguments here but not the other supported 7 (https://github.com/czka/terraform-auth0-modules/blob/reenable_session_lifetimes/auth0-tenant.tf)?

**Comment by czka** [comment]
@aghamyan44 Could you take a look? Seems like these changes are related to your recent refactoring.

**Comment by czka** [comment]
@aramkarapetian @0katrinpetrosyan0 @aghamyan44 
Are you going to develop this module any further? Currently it lacks support for many Auth0 features. I have submitted 3 PRs and have another one in the works (for email and email templates). What are the chances they will be included?

**Comment by czka** [comment]
Please merge it together with another PR: https://github.com/dasmeta/terraform-auth0-modules/pull/18 (I haven't included the respective README updated there, need to be added). These 2 PRs will cover all currently available Auth0 tenant settings.

**Review by aghamyan44 — APPROVED** [review]

**Pull request merged**